client/lib/titan/get-titan-mail-owner-id.js

```diff
@@ -0,0 +1,3 @@
+export function getTitanMailOwnerId( domain ) {
+	return domain?.titanMailSubscription?.ownedByUserId;
+}
```

Comment:
Not related to your pull request, but it would be great to follow a consistent naming for this product in our code: sometimes we refers to it as `TitanMail` and sometimes as just `Titan` (even in this `client/lib/titan` folder).